app/components/layouts/header.tsx

```diff
@@ -153,8 +153,24 @@ class Header extends React.PureComponent<IHeaderProps, {}> {
   public render() {
     const { layoutState } = this.props;
 
+    const { routing } = this.props;
+    const locationSearch = routing.location.search;
+    const searchParams = new URLSearchParams(locationSearch);
+    const searchQueryParam = searchParams.get("query");
+
+    let navClassName;
+    if (layoutState.isTop) {
+      navClassName = styles.navbar;
+    } else {
+      navClassName = `${styles.navbar} ${styles.scrolledNavbar}`;
+    }
+
+    if (searchQueryParam === "" || !searchQueryParam) {
```

Comment:
In javascript, the empty string("") means false.

Reply:
So this condition is a redundant code.

Reply:
OK I will replace it to !searchQueryParam